content/reference/endpoints/bookings.md

```diff
@@ -50,14 +50,14 @@ source            | Integer   | Read     | Source's ID related to the Booking |
 inquiry           | Integer   | Read     | Inquiry's ID related to the Booking |
 created_by_id     | Integer   | Read     | Created By's ID related to the Booking | (User, Booking or Application - this is a polymorphic association) |
 created_by_type   | String   | Read      | Created By's Type related to the Booking | (User, Booking or Application - this is a polymorphic association) |
-booking_comments  | Array    | Read      | Array of related Bookings Comments IDs |
```

Comment:
hmm, but technically those are IDs:
```
links: {
  booking_comments: [ID1, ID2],
  booking_fees: [ID1],
  ...
}
```

Reply:
But if you include those - you will get whole objects, right? So technically those are objects, though reduced by default

Reply:
correct, but then that's applicable to all of the ones above:
```
ccount           | Integer | Read       | Account's ID related to the Booking |
rental            | Integer | Read       | Rental's ID related to the Booking |
client            | Integer | Read       | Client's ID related to the Booking |
rental_agreement  | Integer | Read       | RentalAgreement's ID related to the Bookings Fee |
source            | Integer   | Read     | Source's ID related to the Booking |
inquiry           | Integer   | Read     | Inquiry's ID related to the Booking |
created_by_id     | Integer   | Read     | Created By's ID related to the Booking | (User, Booking or Application - this is a polymorphic association) |
created_by_type   | String   | Read      | Created By's Type related to the Booking | (User, Booking or Application - this is a polymorphic association) |
```
the last one should be probably `created_by` for the sake of consistency

Reply:
Indeed, we have `created_by` exposed and don't have those separate fields. Need to double check.

Reply:
btw, these fields are not in `links`, they appear separately if included. `links` always contain ids. But in the table I guess those rows are not about `links`

Reply:
based on type they were intended as Integers, I'm guessing a bit - but it looks more as a link as they are always in response, for proper objects you'd need to `include` them. So this table sounds more like attributes you can expect by default.

Reply:
We can clarify this as well I guess